### PR TITLE
Features/bigger maps

### PR DESCRIPTION
**Pull request by will-wow**
# Goal

Allow for bigger maps

# Solution

1. Set up a `MapMaker` to make it easier to draw new maps
1. Covert existing maps
1. Create some new 13x13 maps (vs. the old 9x9 ones)
1. Make the battle canvas accomodate different board sizes (and also it's now responsive to window resizes)

## Screenshots

This:

```ruby
      MapMaker.make("Carbon Five", [
        "_____xxx_____",
        "____x_3_x____",
        "___x_____x___",
        "__x_______x__",
        "_x___xxxxxxx_",
        "x___________x",
        "x1_________2x",
        "x___________x",
        "_xx__xxx___x_",
        "__xx_____xx__",
        "____xx4xx____",
        "_____xxx_____",
        "______x______",
      ]),
```

Generates this:

Screen Shot 2019-08-09 at 10 37 22 AM

### CODE REVIEW

File: app/assets/css/battles.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.board {
+  width: 100%;
```
Comment (will-wow) [inline app/assets/css/battles.css]: Responsive board

File: app/models/map.rb [hunk 2]
```diff
@@ -20,43 +20,123 @@ def self.for_name(name)
...
+        "_________",
+        "_________",
+        "_________",
+        "____4____",
+      ]),
+      MapMaker.make("Balanced Team", [
+        "____3____",
+        "_________",
+        "_________",
+        "__x___x__",
```
Comment (will-wow) [inline app/models/map.rb]: All the same maps, but now easier to write and read

File: app/models/map.rb [hunk 3]
```diff
@@ -20,43 +20,123 @@ def self.for_name(name)
...
+        "x_x_x_x_x",
+        "_________",
+        "1_x___x_2",
+        "_________",
+        "x_x_x_x_x",
+        "_________",
+        "__x_4_x__",
+      ]),
+      MapMaker.make("X Marks The Spot", [
+        "x_____3_____x",
```
Comment (will-wow) [inline app/models/map.rb]: Also added some new bigger maps

File: app/assets/js/app/board.coffee [hunk 4]
```diff
@@ -49,6 +50,12 @@ class kablammo.Board
       ammos.find("li").removeClass('ammo-full').addClass('ammo-empty')
     else
       ammos.find("li:gt(#{robot.ammo-1})").removeClass('ammo-full').addClass('ammo-empty')
+  
+  # Canvas height can't be percentage based.
+  updateCanvasHeight: => 
+    $board = $('#board');
+    console.log($board.width())
```
Comment (mwynholds) [inline app/assets/js/app/board.coffee]: Should we get rid of this `console.log`?

**Review by mwynholds — CHANGES_REQUESTED** [review]
Just one straggling `console.log` goes away and then I'm LGTM.

File: app/assets/js/app/board.coffee (hunk 4 above)
Comment (will-wow) [reply app/assets/js/app/board.coffee]: Ah shoot, missed that one!

**Review by mwynholds — APPROVED** [review]
LGTM

**Pull request merged**